<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeachingWidget</class>
 <widget class="QWidget" name="TeachingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(43, 17, 13);</string>
  </property>
  <widget class="QPushButton" name="introToGutButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(48, 21, 14);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>An intro to your Gut</string>
   </property>
  </widget>
  <widget class="QPushButton" name="whyGutHealthButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>460</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(48, 21, 14);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Why is Gut Health important?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="howToStayHealthyButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(48, 21, 14);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>How to keep your gut healthy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backToStartButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(48, 21, 14);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>821</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 43, 24);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>681</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>introToGutButton</zorder>
  <zorder>whyGutHealthButton</zorder>
  <zorder>howToStayHealthyButton</zorder>
  <zorder>backToStartButton</zorder>
  <zorder>graphicsView</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
